engine/buff/sc_buff.cpp

```diff
@@ -2390,7 +2397,14 @@ void buff_t::aura_loss()
   {
     if ( !player->is_sleeping() )
     {
-      sim->print_log( "{} loses {}", *player, name_str );
+      if ( sim->log_spell_id )
+      {
+        sim->print_log( "{} loses {}", *player, *this );
+      }
+      else
+      {
+        sim->print_log( "{} loses {}", *player, name_str );
+      }
```

Comment:
```suggestion
      sim->print_log( "{} loses {}", *player, *this );
```

Reply:
The idea was to replace the whole block with this.

formatting *this (the buff_t object) will automatically print either 'Buff X' or 'Buff X (id)'. So there is no need for the if statement here.